### ISSUE TITLE
WPS508 and chained comparisons

### ISSUE THREAD

**Issue by vnmabus** [wemake-services/wemake-python-styleguide#1942]
### What's wrong

The following code raises a WPS508 code (Found incorrect `not` with compare usage):
```python
if not 1.5 < my_num < 3.75:
    ...
```
The alternative would be:
```python
if my_num <= 1.5 or my_num >= 3.75:
    ...
```
which is probably harder to understand.

The problem is aggravated for longer chains of comparison, or if `my_num` was a function call (because in that case the second way is not even valid, as the function would be called multiple times).

### How it should be

I think this error should not be raised for chained comparisons.

### Flake8 version and plugins

Irrelevant.

### pip information

Irrelevant.

### OS information

Irrelevant.

**Comment by NChechulin** [comment]
Same here. Left a comment to remind of the issue.

**Comment by blablatdinov** [comment]
@sobolevn it's correct behaviour?

**Comment by sobolevn** [comment]
yes, I prefer a more verbose, but a more readable version.
